### PR TITLE
Adds support for bastion host

### PR DESCRIPTION
**Pull request by amoose**
Why
To permit developers to deploy via capistrano

How
Enable SSH proxy (bastion) host

### CODE REVIEW

File: config/deploy.rb [hunk 1]
```diff
@@ -18,8 +19,20 @@
 set :rails_env, :production
 set :repo_url, 'https://github.com/18F/identity-dashboard.git'
 set :ssh_options, forward_agent: false, user: 'ubuntu'
+set :bastion_user, 'ubuntu'
 set :tmp_dir, '/tmp'
 
+set :ssh_options do
+  ssh_command = "ssh -A #{fetch(:bastion_user)}@#{fetch(:bastion_host)} -W %h:%p"
+    {
```
Comment (zachmargolis) [inline config/deploy.rb]: the linters are grumpy here about indendation

File: config/deploy.rb [hunk 2]
```diff
@@ -18,8 +19,20 @@
...
+    proxy: Net::SSH::Proxy::Command.new(ssh_command),
+    user: 'ubuntu',
```
Comment (zachmargolis) [inline config/deploy.rb]: should we reuse `fetch(:bastion_user)` here?

File: config/deploy.rb [hunk 3]
```diff
@@ -18,8 +19,20 @@
 set :rails_env, :production
 set :repo_url, 'https://github.com/18F/identity-dashboard.git'
 set :ssh_options, forward_agent: false, user: 'ubuntu'
+set :bastion_user, 'ubuntu'
 set :tmp_dir, '/tmp'
 
+set :ssh_options do
+  ssh_command = "ssh -A #{fetch(:bastion_user)}@#{fetch(:bastion_host)} -W %h:%p"
```
Comment (zachmargolis) [inline config/deploy.rb]: is there a way to override the bastion user on this line? I thought we were supposed to log in to the bastion as ourselves

File: config/deploy.rb (hunk 2 above)
Comment (amoose) [inline config/deploy.rb]: no, this is not the same user. jumphost <-> apps_host is as `ubuntu`

Comment (zachmargolis) [inline config/deploy.rb]: got it, thanks!

File: config/deploy.rb (hunk 3 above)
Comment (amoose) [inline config/deploy.rb]: I will default it to ubuntu, but yes, I'll make it flexible for higher envs that require we auth via PIV 👍

**Comment by amoose** [comment]
Tested in `qa` 🏁 👍

**Pull request merged**